framework/include/systems/NonlinearSystemBase.h

```diff
@@ -819,6 +841,7 @@ class NonlinearSystemBase : public SystemBase, public PerfGraphInterface
   ///@{
   /// Kernel Storage
   MooseObjectTagWarehouse<KernelBase> _kernels;
+  MooseObjectWarehouse<HDGKernel> _hybridized_kernels;
```

Comment:
This is more of a question based on what I see for finite volume objects. Which one is preferred:
1. Using object warehouses forevery types
2. Putting them into TheWarehouse
I started using the thewarehouse on a different PR, but if (1) is preferred I'll just transition to using object warehouses. And maybe we could do the same for FV too.